### DISCUSSION TITLE
🔗 Container typography settings (as per the previous section / columns), specifically WYSIWYG widget link colors (Duplicate of #20529)

### DISCUSSION THREAD

**Discussion by sf-steve** [elementor/elementor#21638] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The previous container / column layout mode had a typography settings section, that allowed for styling everything within that block.
This was particularly useful for link state colors within WYSIWYG widgets. Given a section design with a dark background, on an otherwise light design site, global settings (theme or Elementor) are unsuitable.

This was easily managed with container settings in previous versions, and was used a lot with alternating dark/light designs etc.
The move to flexbox containers seems to have removed this setting, meaning custom css is required. Whilst not difficult to do, finding global color css var names is tricky, and hard coded colors (and custom css in general) make maintenance harder.
Its also completely unsuitable for end users (clients) and undermines one of the key selling points of Elementor for them (being able to make on-brand design changes without a developer).

Has this setting been moved somewhere we have missed, or is css indeed the only solution currently?

### Describe the solution you'd like

Adding container typography setting

### Describe alternatives you've considered

css

**Comment by pumadog** [comment]
Yes, I deeply miss the settings as well. Can't be that we need custom CSS for this basic function from now on. My request: https://github.com/elementor/elementor/discussions/20529

An alternative solution could be additional global link styles.

**Comment by jontng** [comment]
This needs to comeback.

Also. The background color in the Container widget needs to be a responsive setting.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here [#20529](https://github.com/orgs/elementor/discussions/20529).

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards,

Nick
